integreat_cms/cms/views/push_notifications/push_notification_sender.py

```diff
@@ -86,19 +102,14 @@ def send_pn(self, pnt):
         :return: Response of the :mod:`requests` library
         :rtype: ~requests.Response
         """
-        if settings.DEBUG:
-            region = Region.objects.get(
-                slug=settings.TEST_REGION_SLUG
-            )  # Testumgebung - prevent sending PNs to actual users in development
-        else:
-            region = self.push_notification.region
         payload = {
-            "to": f"/topics/{region.slug}-{pnt.language.slug}-{self.push_notification.channel}",
+            "to": f"/topics/{self.region.slug}-{pnt.language.slug}-{self.push_notification.channel}",
             "notification": {"title": pnt.title, "body": pnt.text},
             "data": {
+                "news_id": str(pnt.id),
```

Comment:
@klinzo Is this what you had in mind with https://github.com/digitalfabrik/integreat-cms/issues/760?

Also, can we use this opportunity to get some consistency into the data? We somehow mix `camelCase` and `snake_case` here and use `blog_id` and `city` for regions... Or are these fields needed exactly like this for backwards compatibility?

Reply:
Yes, that is definitely what I had in mind with #760, is this the same id that will later be used as `id` in this endpoint: https://github.com/digitalfabrik/cms/wiki/REST-APIv3-Documentation#get-messages-for-channel

Feel free to adjust the attributes as needed, we don't use those yet since they are not available in wordpress afaik, right? So you can definitely remove/change `blog_id` and change `lanCode` to `language_code`. If you want you can also change `city` to `city_code`.

Reply:
Alright, thanks! :+1:

> at least if this is the same id that will later be used as `id` in this endpoint

Yes, it is :ok_hand: